spec/features/course/homepage_spec.rb

```diff
@@ -6,10 +6,23 @@
 
   with_tenant(:instance) do
     let(:course) { create(:course, :opened) }
-    let(:achievement_feed_notification) do
-      achievement = create(:course_achievement)
+    let(:feed_notifications) do
```

Comment:
though I can imagine that @allenwq doesn't find it that useful because the let block below already does the bare minimum.

What did you guys discuss?